docs/guides/install-qiskit.mdx

```diff
@@ -83,7 +87,7 @@ Whether you will work locally or in a cloud environment, the first step for all
            </TabItem>
          </OperatingSystemTabs>
 
-1. [Install pip](https://pip.pypa.io/en/stable/installation/). Pip is a Python package manager that you use to install Qiskit and other Python packages. Use `pip list` to see what is in your virtual environment.
+1. [Install pip](https://pip.pypa.io/en/stable/installation/) if you don't use a Python environment. Pip is a Python package manager that you use to install Qiskit and other Python packages. Use `pip list` to see what is in your virtual environment. If you use a Python environment, pip is already installed in the environment.
```

Comment:
This is not always true. There are times where you're using normal Python rather than Anaconda etc, and pip still isn't installed. We need to hedge more.

```suggestion
1. [Install pip](https://pip.pypa.io/en/stable/installation/) if you don't use a Python environment. Pip is a Python package manager that you use to install Qiskit and other Python packages. Use `pip list` to see what is in your virtual environment. If you use a Python environment, pip is usually already installed in the environment.
```